examples/e-commerce/src/app.css

```diff
@@ -103,12 +103,6 @@ h2 {
   margin: 4rem 0;
 }
 
-/* Styles the SFFV highlightings */
-
-mark {
```

Comment:
Unfortunately this CSS class is still needed for now because the results in SFFV are using plain `mark`, not `mark.ais-Highlighted` (the issue comes from the InstantSearch connector).

We can keep the CSS class for now and also add `em` because of the bad support of `ais-highlight` in Angular InstantSearch for now.

![image](https://user-images.githubusercontent.com/6137112/59665564-73d4a500-91b3-11e9-99a0-c65ea38880db.png)

Reply:
what does SFFV stand for?

Reply:
_SFFV = Searching For Facet Values_.

That's what we call the search items within a refinement list.

Reply:
Okay I'll add it back and add `em` as well.
But I don't understand how this issue comes from the IS connector. Isn't it just a bad support of `ais-highlight` in Angular-IS?
If I change the implementation using `highlight` function from `instantsearch.js/es/helper`, I think it's going to be alright. Please correct me if I'm wrong.

Reply:
We could either fix this at the [connector level](https://github.com/algolia/instantsearch.js/blob/af98a46543509d5101586242b4be14a64ed2762d/src/connectors/refinement-list/connectRefinementList.js#L153) that passes the highlighted version of the attribute value, or at the [component level](https://github.com/algolia/angular-instantsearch/blob/106df9ef102e7f701fcc3409a75ebc586b6bbf8f/src/refinement-list/refinement-list.ts) indeed!

Reply:
Ahah I've got what you mean. So have we decided which way to go? or not yet?

Reply:
We haven't discussed that formally yet.

Reply:
Ah okay, thanks!